### DISCUSSION TITLE
Menu Option: Hide Reset Button

### DISCUSSION THREAD

**Discussion by aZtOcKdOg** [vAmigaWeb/vAmigaWeb#179] [Ideas]
If we could hide the reset button as an option in the settings menu that would be welcome! Been a few accidental resets and no savestate oops

It really works great... have been throwing a few productivity / os things at it purely using touch mouse really great super smooth

**Comment by mithrendal** [comment]
Hi @aZtOcKdOg that’s a good find … 

Question: wouldn’t it be better leave the button where it is and to bring a popup „by resetting you might loose unsaved Amiga work. Are you still willing to reset ?“ with a yes/no choice… when one hits reset ?

**Comment by aZtOcKdOg** [comment]
That sounds like a good idea for confirmation dialog to appear for the reset button! You don't think it could live one click in to settings and present near the 'Inserted Devices' (DF0-DF3 / DH0-DH3) or I guess there is limited real estate in that area... 

A confirmation for reset will be very great, the ability to hide would be excellent but then, for me, reset isn't used as much as the hamburger icon is. So I would welcome whatever change is: 1. Readily achievable 2. A solid UI / UX enhancement 

Thanks @mithrendal o/

**Comment by mithrendal** [comment]
<img width="1210" alt="image" src="https://github.com/vAmigaWeb/vAmigaWeb/assets/17108995/673f578b-a954-4b5f-8470-c5761e3d0797">

added the reset confirmation dialog to https://vamigaweb.github.io
